static/sass/_pattern_navigation.scss

```diff
@@ -1,23 +1,103 @@
 @mixin anbox-p-navigation {
-  .p-navigation__banner {
-    background-color: $ubuntu-orange;
-  }
-
-  .p-navigation .p-navigation__row {
+  .p-navigation {
```

Comment:
We have a convention where we don't concatenate classes in this way because it makes them harder to search for or keep track of in a large stylesheet.

See: https://canonical-web-and-design.github.io/practices/coding/stylesheets.html#naming-convention

Reply:
Ahh yes I totally forgot about this! Thanks Steve I'll change it now